app/controllers/patients_controller.rb

```diff
@@ -21,10 +21,12 @@ def index
 
   def new
     @patient = Patient.new
+    @patient_nav = false
     respond_with(@patient)
```

Comment:
So I think the syntax for above is more like:
``` respond_with(@patient, locals: { show_nav: false }) ```

However, I can't get this to work still. It doesn't seem to be available in the local_assigns hash. i.e. it doesn't carry over as false and displays anyway.